modules/tensor_mechanics/src/materials/PowerLawSoftening.C

```diff
@@ -0,0 +1,49 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "PowerLawSoftening.h"
+
+#include "MooseMesh.h"
+
+template <>
+InputParameters
+validParams<PowerLawSoftening>()
+{
+  InputParameters params = validParams<SmearedCrackSofteningBase>();
+  params.addClassDescription("Softening model with an abrupt stress release upon cracking. This "
+                             "class is intended to be used with ComputeSmearedCrackingStress.");
+  params.addRequiredRangeCheckedParam<Real>(
+      "stiffness_reduction",
```

Comment:
No default value?

Reply:
It used to be 1/3 by default.  I could be swayed to make it have a default, but it seemed a little odd to me to have something like that be a default since it's a pretty fundamental property of the model (kind of like a modulus of elasticity -- you wouldn't have that have a default value).

Reply:
Good point.  I'm thinking Bison user and not general user.